series/series_test.go

```diff
@@ -1165,3 +1285,49 @@ func TestSeries_IsNaN(t *testing.T) {
 		}
 	}
 }
+
+func TestSeries_Empty(t *testing.T) {
+	tests := []struct {
+		name string
+		s    Series
+		want Series
+	}{
+		{"Series Empty", Series{Name: "test", t: String}, New([]int{}, String, "test")},
+	}
+	for i, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Please, try to be consistent with the way the rest of the tests are written. Don't use `t.Run`. Also, this should be a separate PR. (Same thing with TestSeries_Append)